test/integration/smoke/test_deploy_vm_extra_config_data.py

```diff
@@ -130,6 +124,10 @@ def setUp(self):
         updateConfigurationResponse = self.apiclient.updateConfiguration(updateConfigurationCmd)
         self.debug("updated the parameter %s with value %s" % (
             updateConfigurationResponse.name, updateConfigurationResponse.value))
+        self.cleanup = []
```

Comment:
it is empty by default, right ?

Reply:
yes, it is in this file only filled by deploy_vm()